src/controller/TypedReadCallback.h

```diff
@@ -80,7 +80,9 @@ class TypedReadAttributeCallback final : public app::ReadClient::Callback
         VerifyOrDie(!aPath.IsListItemOperation());
 
         VerifyOrExit(aStatus.IsSuccess(), err = aStatus.ToChipError());
-        VerifyOrExit(aPath.mClusterId == mClusterId && aPath.mAttributeId == mAttributeId, err = CHIP_ERROR_SCHEMA_MISMATCH);
+        VerifyOrExit((aPath.mClusterId == mClusterId || kInvalidClusterId == mClusterId) &&
+                         (aPath.mAttributeId == mAttributeId || kInvalidAttributeId == mAttributeId),
+                     err = CHIP_ERROR_SCHEMA_MISMATCH);
```

Comment:
This doesn't make sense.  TypedReadAttributeCallback is tied to a specific type, derived from a specific attribute ID.  You can't use it for a "wildcard" request...